src/event-handler/event-channel-manager.ts

```diff
@@ -0,0 +1,125 @@
+import Ain from '../ain';
+import EventHandler from './index';
+import { WebSocket } from 'ws';
+import {
+  EventChannelMessageTypes,
+  EventChannelMessage,
+  BlockchainEventTypes,
+} from '../types';
+import EventFilter from './event-filter';
+
+export default class EventChannelManager {
```

Comment:
As this module handles only 1 channel, what about 'EventChannelClient'?

Reply:
Thanks! Done